lib/main.dart

```diff
@@ -520,10 +523,21 @@ class _ProvenanceWalletAppState extends State<ProvenanceWalletApp> {
       () => PriceService(),
     );
 
-    get.registerSingleton<WalletConnectService>(DefaultWalletConnectService());
-
     final accountService = get<AccountService>();
     final remoteNotificationService = get<RemoteNotificationService>();
+    final walletConnectQueueService = get<WalletConnectQueueService>();
+
+    get.registerSingleton<WalletConnectService>(
+      DefaultWalletConnectService(
+        keyValueService: keyValueService,
+        accountService: accountService,
+        connectionFactory: walletConnectionFactory,
+        notificationService: remoteNotificationService,
+        queueService: walletConnectQueueService,
+        transactionHandler: transactionHandler,
+        localAuthHelper: authHelper,
```

Comment:
Not only does this improve visibility, I love how all but one of those services was already retrieved via get in this scope. 👍🏻